pjsip-apps/src/pjsua/pjsua_app_config.c

```diff
@@ -1601,7 +1601,7 @@ static void default_config()
     pjsua_app_config *cfg = &app_config;
 
     pjsua_config_default(&cfg->cfg);
-    pj_ansi_sprintf(tmp, "PJSUA v%s %s", pj_get_version(),
+    pj_ansi_snprintf(tmp, sizeof(tmp), "PJSUA v%s %s", pj_get_version(),
                     pj_get_sys_info()->info.ptr);
     pj_strdup2_with_null(app_config.pool, &cfg->cfg.user_agent, tmp);
```

Comment:
snprintf code indent all not align  in this file

Reply:
yes, they're intentional to minimize code change (re: git blame)